_sass/components/_buttons.scss

```diff
@@ -80,6 +80,18 @@
   }
 }
 
+.btn-open-in-repl {
+  position: absolute;
+  bottom: 0;
+  padding: 1rem;
+  right: 55px; // size of the button.btn-clipboard
```

Comment:
I'm not a CSS expert. This is causing some issues:
![bug](https://cloud.githubusercontent.com/assets/1493671/21466655/239da9d4-c9d2-11e6-8473-fb5908d01d76.png).

Would it possible to use a flexbox wrapping both buttons?